scripts/workers/worker_util.py

```diff
@@ -85,7 +85,7 @@ def get_challenge_phase_by_pk(self, challenge_pk, challenge_phase_pk):
         response = self.make_request(url, "GET")
         return response
 
-    def update_submission_data(self, data, challenge_pk, submission_pk):
+    def update_submission_data(self, data, challenge_pk):
```

Comment:
Can you make a separate PR for this change as it is not part of `Add tests for worker_util.py`

Reply:
Great! @takitsuse had opened an PR already: https://github.com/Cloud-CV/EvalAI/pull/2611

Reply:
Done here! :+1: